include/itkMGHImageIO.h

```diff
@@ -49,7 +49,7 @@ class MGHIO_EXPORT MGHImageIO:public ImageIOBase
   itkNewMacro(Self);
 
   /** Run-time type information (and related methods). */
-  itkTypeMacro(MGHImageIO, Superclass);
+  itkTypeMacro(MGHImageIO, ImageIOBase);
```

Comment:
Do you have a specific reason for this change? For me, the original version was better.

It is good practice to use `Superclass` instead of keep repeating the actual class name as it makes the code easier to read (developers don't need to scroll up to verify that the superclass name is written there), and it reduces the chances of copy-paste errors.

Reply:
Consistency: all across the ITK code base is done this way, most remotes probably do as well, and the ITK SW Guide states the RTTI to be this way. If `Superclass` is preferred, then IMO the same rationale applies to the `Self` part: why don't we use `Self` for the self class? Again, virtually no instance uses `Self` for the RTTI macro.

Reply:
OK, if it is done like this in most classes then it means that there must be a some specific reason of going into the trouble of spelling out the name of the superclass. @thewtex do you have any insights into this? If indeed there is a reason not to use `Superclass` then this should be changed in all the other 25 classes where `Superclass` used in `itkTypeMacro`.

Reply:
Yes we want to use the actual, literal superclass name instead of `Superclass`. This string is used in a macro which is using the input (although currently not using the superclass input) for run-time type information:

https://github.com/InsightSoftwareConsortium/ITK/blob/8844ee6549ebdc1294a17d471f8a65311440bd74/Modules/Core/Common/include/itkMacro.h#L438

Reply:
https://github.com/InsightSoftwareConsortium/ITK/pull/4342